README.md

```diff
@@ -95,6 +95,13 @@ Executes the `jsdoc -X` command and parses the output into a Javascript object/a
             Required (if <code>files</code> is not set). Documented source code to be processed. If <code>files</code> is also set, this will be ignored. Default: <code>undefined</code>
         </td>
     </tr>
+    <tr>
+        <td><b><code>config</code></b></td>
```

Comment:
Added this in the README to make the PR more complete. As changes are requested this can be updated.